src/frontend/Sider/Behandling/Inngangsvilkår/typer/vilkårperiode.ts

```diff
@@ -45,6 +45,12 @@ export enum SvarJaNei {
     NEI = 'NEI',
 }
 
+export enum BegrunnelsePåkrevd {
+    JA = 'JA',
+    NEI = 'NEI',
+    JA_HVIS_NEI = 'JA_HVIS_NEI',
+}
+
```

Comment:
Kanskje mer forståelig?
```suggestion
export enum BegrunnelseObligatorisk {
    OBLIGATORISK = 'OBLIGATORISK',
    VALGFRI = 'VALGFRI',
    OBLIGATORISK_HVIS_SVAR_NEI = 'OBLIGATORISK_HVIS_SVAR_NEI',
}
```

Reply:
Jeg syntes dette ser bra ut!  

Eventuelt la `BegrunnelseObligatorisk.OBLIGATORISK_HVIS_SVAR_NEI ` hete `BegrunnelseObligatorisk.BETINGET`